packages/api/src/roles/all.ts

```diff
@@ -30,6 +30,11 @@ export const all = (role, actions) => {
     return false;
   };
 
+  const isPrivateFile = (file, _, context) => {
+    if (!file?.isPrivate || file?.meta?.authorId === context?.userId) return true;
```

Comment:
1. isPrivateFile is the wrong name for this function, it should be something like isFileAccessible because it also allows access to public files
2. We have only used the naming 'authorId' for reviews, we should user userId to stay consistent with the rest